Dockerfile

```
@@ -1,6 +1,6 @@
 ### This is a generated file from Dockerfile.in ###
-#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.16)
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
+#@follow_tag(registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:rhel_8_golang_1.18)
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.8 AS builder
```

For me this image does not seem to exist. The only `1.18` image I can pull is the one with `openshift-4.11` in the name:

```suggestion
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11 AS builder
```